gapic/src/main/com/google/gapid/models/ProfileExperiments.java

```diff
@@ -16,21 +16,23 @@
 package com.google.gapid.models;
 
 import com.google.gapid.proto.service.Service;
+import com.google.gapid.proto.service.path.Path;
+
+import java.util.List;
 
 public final class ProfileExperiments {
   public final boolean disableAnisotropicFiltering;
+  public final List<Path.Command> disabledCommands;
```

Comment:
Use an `ImmutableList<Path.Command>` here to maintain `ProfileExperiments`'s immutablility.